moveit2.repos

```diff
@@ -0,0 +1,47 @@
+repositories:
```

Comment:
Add comment at top of this file and the .rosinstall file explaining why we have both. Because I don't understand why

Reply:
Actually, the .repos format is more commonly used in ROS2. We still have the .rosinstall for legacy reasons (i.e. moveit_ci). I think we should remove the rosinstall in the future.

Reply:
`vcstool` seems to support pretty much all features of `wstool`.
It is focused more on the actual content of the workspace (you don't run `init`, instead you just import config files) and offers IMO better usability for workspace-wide git (and others) commands: `branch`, `diff`, `status`, `remotes`, `log`, `pull`, `push`. The command `vcs validate` parses your .repos file and compares the setup with your workspace.
The biggest difference is how workspace files are created or modified. You can't remove single entries from your config file anymore but instead you can directly export your current workspace setup into a new configuration file (or overwrite the existing one).